### PR TITLE
Fix for the prior posterior plot incorrectly switching the bf10 and bf01 label

### PR DESCRIPTION
**Pull request by JTPetter**
Fixes jasp-stats/jasp-issues#796

### CODE REVIEW

File: R/commonsummarystatsttestbayesian.R [hunk 1]
```diff
@@ -361,7 +361,7 @@
     n2                        = 0 ,
     paired                    = FALSE,
     BF10user                  = BFlist[["BF10"]],
-    yAxisLegendRobustnessPlot = BFPlots,
+    yAxisLegendRobustnessPlot = BFPlotsH1H0,
```
Comment (AlexanderLyNL) [inline R/commonsummarystatsttestbayesian.R:364]: @JTPetter Thanks for figuring this out!

@vandenman It seems that this plots H1 above and H0 below if TRUE, whereas it plots H0 above and H1 below if FALSE. Did something change in the development process?

**Review by AlexanderLyNL — CHANGES_REQUESTED** [review]
Just some small style changes.

**Comment by AlexanderLyNL** [comment]
Please also make a PR to the stable branch of jasp-stats/jasp-desktop.

File: R/commonsummarystatsttestbayesian.R (hunk 1 above)
Comment (vandenman) [reply R/commonsummarystatsttestbayesian.R]: nothing changed to the JASPgraphs code in 5 months, so I don't think so. Perhaps this was introduced during the rewrite?

**Pull request merged**